polaris-intellisense/README.md

```diff
@@ -0,0 +1,23 @@
+# `polaris-intellisense` VSCode Extension
+
+:warning: This extension is still an experimental prototype
+
+VSCode intellisense features to support developing with the Polaris Design System.
+
+## Installation
+
+We don't want to publish this extension to the VSCode Extensions Marketplace yet, so we have to install the package directly from the compiled `.vsix` file.
+
+1. (Install the `vsce` package)[https://code.visualstudio.com/api/working-with-extensions/publishing-extension#vsce]
+2. `cd` to the `polaris-intellisense` directory in the project.
+3. From the `polaris-intellisense` directory run the (`vsce package`)[https://code.visualstudio.com/api/working-with-extensions/publishing-extension#packaging-extensions] command in your terminal. After running you should have a new `.vsix` file in the directory (ex. `polaris-intellisense-0.0.1.vsix`)
+4. To install the extension run `code --install-extension EXTENSION_FILENAME.vsix`
+
+```bash
+# example of running install command
+❯ code --install-extension shopify-dev-intellisense-0.0.1.vsix
+```
```

Comment:
```suggestion
```sh
# example of running install command
code --install-extension shopify-dev-intellisense-0.0.1.vsix
```
```

Suggestion leaving off the `❯` character. Makes copy/paste less convenient IMO